traefik/values.yaml

```diff
@@ -410,3 +410,13 @@ securityContext:
 
 podSecurityContext:
   fsGroup: 65532
+
+# Topology spread constraints rely on node labels to identify the topology domain(s) that each Node is in.
+# Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
+topologySpreadConstraints: []
+#   - maxSkew: 1
+#     topologyKey: failure-domain.beta.kubernetes.io/zone
```

Comment:
The current label is deprecated, I suggest to use the following
```suggestion
#     topologyKey: topology.kubernetes.io/zone
```
https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone